packages/workspace/src/helpers/utils.ts

```diff
@@ -1,14 +1,31 @@
 import { ConfigurationService, ConfigurationServiceHttp } from '@capsulajs/capsulajs-configuration-service';
 import { API } from '..';
 import * as INTERNAL_TYPES from './types';
-import { bootstrapComponentError, bootstrapServiceError } from './const';
+import {
+  getLoadingServiceError,
+  getLoadingComponentError,
+  getBootstrapServiceError,
+  getInitComponentError,
+  getBootstrapComponentError,
+} from './const';
 
 export const getConfigurationService = (token: string): ConfigurationService<API.WorkspaceConfig> =>
   new ConfigurationServiceHttp(token);
 
+export const dynamicImport = (path: string) => import(path).then((module) => module.default);
```

Comment:
Is this implicitly returning a Promise ?

Reply:
explicitly - no need to specify it